src/uct/base/uct_iface.c

```diff
@@ -857,6 +857,92 @@ void uct_iface_get_local_address(uct_iface_local_addr_ns_t *addr_ns,
     }
 }
 
+void uct_iface_set_sys_dev(const char *dev_name, const char *sysfs_path,
```

Comment:
Maybe move this func and the next one to UCS?

Reply:
won't it complicate topo API?

Reply:
IMO also it should be in ucs/topo since it deals with sysfs paths

Reply:
return sys_dev instead of void